client/platform/desktop/backend/native/common.ts

```diff
@@ -830,10 +848,21 @@ async function _importTrackFile(
 
     const newPath = npath.join(projectDirAbsPath, npath.basename(newFileName));
 
-    await fs.copy(
-      userTrackFileAbsPath,
-      newPath,
-    );
+    if (jsonMeta.originalVideoFile && await nistSerializers.confirmNistFile(userTrackFileAbsPath)) {
+      const trackData = await nistSerializers.loadNistFile(userTrackFileAbsPath);
```

Comment:
Where is `fullFrameBounds` passed?  Looks like nothing uses that argument.

Reply:
The CLI version uses it when doing conversions, in all other instances it defaults to 1080p-ish size.  I was hesitant about making the system do another `checkMedia` call on import for each NIST file or storing it in the metadata.  For right now I'm going to default to the full frame size given that the source NIST videos typically are that resolution.